CHANGELOG.md

```diff
@@ -23,19 +27,47 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added Echo icon
 - Added poorConnection property to DeskPhone icon
 - Added optional 'id' prop for ui components
+- Added optional `timestamp` prop in ChatBubble
+- Added forwardRef for ChatBubbleContainer
+- Added optional img to MessageAttachment
 - Added a classname to PopOverMenu component for styling access
+- Added forwardRef for Textarea
+- Added `useDevicePermissionStatus` hook as an exported component from the library.
 - Added a "dismissible" prop to Modal to optionally allow persistent modals
 - Added ZoomIn and ZoomOut icons
+- Added optional `timestamp` prop in ChatBubble
+- Added forwardRef for ChatBubbleContainer
+- Added optional img to MessageAttachment
+- Added a classname to PopOverMenu component for styling access
+- Added forwardRef for Textarea
 - Added style variants to Caution icon
 
 ### Changed
 
 - Update Jest major version
+- Changed RosterHeader 'title' prop to all for elements as well as strings
+- Changed senderName to optional in ChatBubble
+- Moved children inside of a div in ChatBubble
+- Changed `MeetingManager` to strictly enforce `DevicePermissionStatus` type.
+- Update `realtimeUnsubscribeFromVolumeIndicator` interface to also accept a callback param.
+- Refactored NavBar to allow static width
+- Refactored NavBarItem to use IconButton directly
+
+### Removed
+
+- Removed content in ChatBubble
+- Removed showName in ChatBubble
 - Change control bar theme opacity to 1
+- Changed RosterHeader 'title' prop to all for elements as well as strings
+- Changed senderName to optional in ChatBubble
+- Moved children inside of a div in ChatBubble
+- Changed `MeetingManager` to strictly enforce `DevicePermissionStatus` type.
```

Comment:
Duplicates from 10lines above

Reply:
Good catch. I had to comb through these changelog conflicts like 10 times.